scheduler/bin/run-local.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+# Usage: ./bin/run-local.sh
+# Runs the cook scheduler locally.
+
+# Defaults (overridable via environment)
+: ${COOK_DATOMIC_URI="datomic:mem://cook-jobs"}
+: ${COOK_FRAMEWORK_ID:=cook-framework-$(date +%s)}
+: ${COOK_NREPL_PORT:=${2:-8888}}
+: ${COOK_PORT:=${1:-12321}}
+: ${MASTER_IP:="127.0.0.2"}
```

Comment:
@shamsimam why does this default to 127.0.0.2?

Reply:
I have ZK running on 127.0.0.1, master on 127.0.0.2 and the workers on 3/4. I used Waiter's add loopback script to get all these IPs working locally.